source/assets/css/_imports/des-guide.less

```diff
@@ -0,0 +1,346 @@
+.start-guide {
+    .des-guide {
+        text-align: center;
+
+        img {
+            margin-top: 0;
+            width: 100px;
+        }
+    }
+
+    h1:first-of-type {
+        color: #142432;
+        font-size: 33px;
+        font-weight: 600;
+        margin-bottom: 0;
+        padding-bottom: 0;
+    }
+
+    .info.guide {
+        font-weight: 600;
+    }
+
+    .anchorjs-icon:before {
+        content: '';
+    }
+}
+
+/*------- START & END COLUMN -------*/
+.des-guide--opener,
```

Comment:
You're using two classes for the same purpose. This usually indicates code smell. It would be better to add a new class which can be assigned to both elements.

Reply:
check.
use now: guide--container